packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -964,7 +972,8 @@ const ReactDataGrid = createReactClass({
             onViewportKeyup={this.onKeyUp}
             onViewportDragStart={this.onDragStart}
             onViewportDragEnd={this.handleDragEnd}
-            onViewportDoubleClick={this.onViewportDoubleClick}
+            onViewportClick={this.deselect}
+            onViewportDoubleClick={this.deselect}
```

Comment:
@malonecj Clicking outside a row or cell will now deselect the currently selected cell. Alternatively we could get rid of `onViewport(Double)Click` all together since we already have a `onCellDoubleClick`.